runtime/onert/core/include/loader/CircleLoader.h

```diff
@@ -14,20 +14,20 @@
  * limitations under the License.
  */
 
-#ifndef __CIRCLE_CIRCLE_LOADER_H__
-#define __CIRCLE_CIRCLE_LOADER_H__
+#ifndef __ONERT_LOADER_CIRCLE_LOADER_H__
+#define __ONERT_LOADER_CIRCLE_LOADER_H__
```

Comment:
What is the rule for inserting an underscore within a file name? i.e. why isn't this `__ONERT_LOADER_CIRCLELOADER_H__`?

Reply:
There are no set rules, but by convention, if it starts with an uppercase letter, it is followed by an underscore. For example, if it's CircleLoader, it's considered as CIRCLE_LOADER.